system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -42,6 +43,43 @@
                            (map #(select-keys % [:concept-id :extra-fields :metadata])))]
     (#'jobs/process-subscriptions (system/context) subscriptions)))
 
+
+(deftest subscription-ingest-no-subscriber-id-test
+  (let [coll1 (data-core/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection
+                {:ShortName "coll-no-id"
+                 :EntryTitle "entry-title-no-id"})
+               {:token "mock-echo-system-token"})]
+    (testing "ingest on PROV1, with no subscriber-id supplied"
```

Comment:
Have we considered the use case where an admin user (e.g. Mitch) tries to create a subscription for a regular user (e.g. Mark) and there is a subscriber-id `Mark` in the subscription, but no email? What do we do in this case?

Reply:
This has been considered - tests now cover this use case explicitly